service/routes.py

```diff
@@ -1,455 +1,425 @@
+POST /inventory - Creates a new Inventory record in the database
+PUT /inventory/{product_id}/{condition} - Updates an Inventory object record in the database
+PUT /inventory/{product_id}/{condition}/active - Change an item's update status to enabled
+DELETE /inventory/{product_id}/{condition/active - Change an item's update status to disabled
+DELETE /inventory/{product_id}/{condition} - Deletes an Inventory object record in the database
 """
-# Standard library imports
-from enum import Enum
-
-# Third-party imports
 
-from flask import jsonify, request, abort
+from flask import jsonify
+from flask_restx import Resource, fields
 from sqlalchemy.exc import IntegrityError
-
-# Local imports
 from service.models import Inventory, Condition, UpdateStatusType
 from service.common import status  # HTTP Status Codes
-from service.utilities import check_content_type, check_condition_type
-
-# Import Flask application
-from . import app
-
+from service.utilities import check_condition_type
+from . import app, api
 
-class FilterType(Enum):
-    """Enumeration of filter types"""
-
-    CONDITION = (0,)
-    RESTOCK = (1,)
-
-    # This is the last value in the enum and is meant to be a default value. Nothing should come after this
-    NONE = 2
 
+######################################################################
+# Configure the Root route before OpenAPI
+######################################################################
+@app.route("/")
+def index():
+    """Index page"""
+    return app.send_static_file("index.html")
 
-# end enum FilterType
 
 ############################################################
 # Health Endpoint
 ############################################################
 @app.route("/health")
 def health():
     """Health Status"""
-    return {"status": 'OK'}, status.HTTP_200_OK
+    return {"status": "OK"}, status.HTTP_200_OK
 
 
-######################################################################
-# GET INDEX
-######################################################################
-@app.route("/")
-def index():
-    """Root URL response"""
-    return app.send_static_file("index.html")
-
-
-######################################################################
-#  R E S T   A P I   E N D P O I N T S
-######################################################################
-
-
-######################################################################
-# ENABLE UPDATES OF A PRODUCT ID
-######################################################################
-@app.route(
-    "/inventory/<string:_product_id>/<string:_condition>/active", methods=["PUT"]
+# Define the model so that the docs reflect what can be sent
+create_model = api.model(
+    "Inventory",
+    {
+        "product_id": fields.Integer(
+            required=True, description="The product ID of the Inventory"
+        ),
+        "condition": fields.String(
+            required=True,
+            enum=Condition._member_names_,
```

Comment:
Take a look at the comment I left in tdd.yml that references this line. You need to specify the "._member_names_" field or else you'll get an API load error